blue_description/README.md

```diff
@@ -1,4 +1,4 @@
 # Gazebo Models
 
-The models included within this package have been ported from the
+Many of the models included within this package have been ported from the
```

Comment:
This is a pretty basic README. I would like to see a more descriptive README defining each of the respective models and providing proper credit.

Reply:
I think that this should be included in #31